torchci/clickhouse_queries/strict_lag_sec/query.sql

```diff
@@ -0,0 +1,34 @@
+WITH master as (
+    SELECT
+        push.head_commit.timestamp as master
+    FROM
+        push
+    WHERE
+        push.ref = {head: String }
+        AND push.repository.owner.name = 'pytorch'
```

Comment:
Nit: Should we add a parameter for owner here too instead of fixing it as pytorch?  It's a trivial fix

Reply:
Done, its not a param for the rockset version tho, but it doesn't get used anywhere so its fine imo